frontend/main.py

```diff
@@ -253,18 +261,41 @@ def delete_correction(queue_name, correction_id, version):
         correction_id)}, ExpressionAttributeValues={':v1': version}, ConditionExpression='version = :v1')
 
 
+def reset_correction(correction, reset_fields):
```

Comment:
Let's not make the reset field configurable. Let's always reset these fields.
Please also move the helper method to the bottom.

Reply:
@hasankhan do we want to reset even the 'id' while moving to different queue? That is why I explicitly took the list of fields to rest.